pyproject.toml

```diff
@@ -26,7 +27,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.6"
```

Comment:
Does this no longer break `black`?

Reply:
Black is a development dependency, even is this is the case, it should not matter.